code/renderers/server/src/preset.ts

```diff
@@ -1,36 +1,30 @@
 import fs from 'fs-extra';
 import yaml from 'yaml';
-import { toId } from '@storybook/csf';
-import type { StaticMeta } from '@storybook/csf-tools';
-import type { IndexedStory, StoryIndexer, StorybookConfig } from '@storybook/types';
+import type { StorybookConfig, Tag, StoryName, ComponentTitle } from '@storybook/types';
 
-export const storyIndexers: StorybookConfig['storyIndexers'] = (existingIndexers) => {
-  const serverIndexer: StoryIndexer['indexer'] = async (fileName) => {
-    const json = fileName.endsWith('.json')
-      ? await fs.readJson(fileName, 'utf-8')
-      : yaml.parse((await fs.readFile(fileName, 'utf-8')).toString());
-    const meta: StaticMeta = {
-      title: json.title,
-    };
-    const stories: IndexedStory[] = json.stories.map((story: { name: string }) => {
-      const id = toId(meta.title, story.name);
-      const { name } = story;
-      const indexedStory: IndexedStory = {
-        id,
-        name,
-      };
-      return indexedStory;
-    });
-    return {
-      meta,
-      stories,
-    };
-  };
-  return [
-    {
-      test: /(stories|story)\.(json|ya?ml)$/,
-      indexer: serverIndexer,
-    },
-    ...(existingIndexers || []),
-  ];
+type FileContent = {
+  title: ComponentTitle;
+  tags?: Tag[];
+  stories: { name: StoryName; tags?: Tag[] }[];
 };
+
+export const indexers: StorybookConfig['indexers'] = (existingIndexers) => [
```

Comment:
So this changed from `storyIndexers` to `indexers`, right.
Maybe I'm missing it in this PR, but I'd expect this change to be actually used somewhere, was that already done in another PR?

Might this be marked as experimental in any way?